config/dev1/sys.config

```diff
@@ -65,6 +65,8 @@
   ]},
 
   {setup, [
-           {abort_on_error, true}
+           {abort_on_error, true},
+           {data_dir, "data"},
+           {log_dir, "log"}
```

Comment:
How does this hook with the code?

Reply:
@lucafavatella The `setup` API, `setup:data_dir()` and `setup:log_dir()` will read those values, relative to current working dir, then derive the abspath for each and update the application environment. Afterwards, the above mentioned functions will return fully qualified paths to the `epoch/data` and `epoch/log` directories.

Currently, we don't make use of the `log_dir` function, but it should still return the correct path to our actual log directory, if at all possible.